format_spec/array_file_hierarchy.md

```diff
@@ -22,23 +25,38 @@ my_array                                # array folder
           |_ <timestamped_name>.con     # consolidated commits file
           |_ ...
           |_ <timestamped_name>.ign     # ignore file for consolidated commits file
-    |_ __fragment_meta                  
-          |_ <timestamped_name>.meta    # consol. fragment meta file
-          |_ ...                  
+    |_ __fragment_meta                  # consolidated fragment metadata folder
+          |_ <timestamped_name>.meta    # consolidated fragment meta file
+          |_ ...
     |_ __meta                           # array metadata folder
     |_ __labels                         # dimension label folder
-
+    |_ <timestamped_name>               # legacy fragment folder
+          |_ ...
+    |_ <timestamped_name>.ok            # legacy fragment write file
+    |_ <timestamped_name>.meta          # legacy consolidated fragment meta file
+    |_ __array_schema.tdb               # legacy array schema file
 ```
 
 Inside the array folder, you can find the following:
 
-* [Array schema](./array_schema.md) folder `__schema`.
-* Inside of a fragments folder, any number of [fragment folders](./fragment.md) [`<timestamped_name>`](./timestamped_name.md).
-* Inside of a commit folder, an empty file [`<timestamped_name>`](./timestamped_name.md)`.wrt` associated with every fragment folder [`<timestamped_name>`](./timestamped_name.md), where [`<timestamped_name>`](./timestamped_name.md) is common for the folder and the WRT file. This is used to indicate that fragment [`<timestamped_name>`](./timestamped_name.md) has been *committed* (i.e., its write process finished successfully) and it is ready for use by TileDB. If the WRT file does not exist, the corresponding fragment folder is ignored by TileDB during the reads.
-* Inside the same commit folder, any number of [delete commit files](./delete_commit_file.md) of the form [`<timestamped_name>`](./timestamped_name.md)`.del`.
-* Inside the same commit folder, any number of [update commit files](./update_commit_file.md) of the form [`<timestamped_name>`](./timestamped_name.md)`.upd`.
-* Inside the same commit folder, any number of [consolidated commits files](./consolidated_commits_file.md) of the form [`<timestamped_name>`](./timestamped_name.md)`.con`.
-* Inside the same commit folder, any number of [ignore files](./ignore_file.md) of the form [`<timestamped_name>`](./timestamped_name.md)`.ign`.
-* Inside of a fragment metadata folder, any number of [consolidated fragment metadata files](./consolidated_fragment_metadata_file.md) of the form [`<timestamped_name>`](./timestamped_name.md)`.meta`.
+* Inside of a `__schema` folder, any number of [array schema files](./array_schema.md) [`<timestamped_name>`](./timestamped_name.md).
+  * Note: the name does _not_ include the format version.
```

Comment:
```suggestion
  * **Note**: the name does _not_ include the format version.
```

Reply:
Done and also updated a similar usage in `metadata.md`.